CONTRIBUTING.md

```diff
@@ -268,6 +277,15 @@ See also: #456, #789
 Signed-off-by: Some Dev <somedev@example.com>
 ```
 
+To make any changes to your existing git commit message, run the following command in your terminal:
```

Comment:
Change the section after the comma to: `make sure you are on your development branch and run the following command in your terminal`

Reply:
On it